lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetReader.java

```diff
@@ -623,4 +624,17 @@ private void validateWrite(java.util.function.Predicate<ParquetWriteValidation>
             throw new ParquetCorruptionException(dataSource.getId(), "Write validation failed: " + messageFormat, args);
         }
     }
+
+    private static boolean arrayEqualsReversed(String[] path, String[] otherPath)
```

Comment:
path -> a
otherPath -> b

(this makes the message more abstract and thus easier to understand)